src/main/java/picard/illumina/BasecallsConverter.java

```diff
@@ -339,9 +344,45 @@ protected void setTileLimits(final Integer firstTile, final Integer tileLimit) {
         }
     }
 
+    protected String maybeDemultiplex(ClusterData cluster, Map<String, BarcodeMetric> metrics, BarcodeMetric noMatch) {
```

Comment:
```suggestion
    /** If demultiplexing is to be performed, identify and return the barcode the cluster is assigned to. Otherwise returns null. */
    protected String maybeDemultiplex(ClusterData cluster, Map<String, BarcodeMetric> metrics, BarcodeMetric noMatch) {
```